scrapli_community/alcatel/aos/alcatel_aos.py

```diff
@@ -0,0 +1,17 @@
+"""scrapli_community.alcatel.aos.alcatel_aos"""
+
+from scrapli_community.alcatel.aos._async import default_async_on_close
+from scrapli_community.alcatel.aos.sync import default_sync_on_close
+
+SCRAPLI_PLATFORM = {
+    "driver_type": "generic",
+    "defaults": {
+        # Prompt can be anything, but best practice is to end with > or #
+        "comms_prompt_pattern": r"^[a-zA-Z0-9.\-_@\/:]{1,63}[>|#]\s*",
+        "sync_on_open": None,
+        "async_on_open": None,
+        "sync_on_close": default_sync_on_close,
+        "async_on_close": default_async_on_close,
+        "transport": "ssh2",
```

Comment:
any reason to explicitly set ssh2 as transport? the whole point of community is so folks can have stuff pre canned to what they need/want so I'm totally good with it, just curious!

Reply:
Paramiko also works, but I though ssh2 to be the most performant option:
https://parallel-ssh.org/post/parallel-ssh-libssh2/

Reply:
cool. yeah any of the transports *should* work, but ssh2 is for sure fast! 

Also, heads up: there is no wheel for Darwin for 3.10 yet so just keep that in mind! Lastly -- ssh2 is another requirement for users who want to use this platform w/ defaults (whereas there are literally *no* requirements for `system` transport) -- I'm good with it if you are, just wanted to call it out!